### PR TITLE
GP2-2936 Dac Custom Controls 04

### PR DESCRIPTION
**Pull request by seijihg**
- Added Aria and role for <div> radiogrpitem for JAWS recognition. Tested with ChromeVox on Mac

Should recognise when radio is disabled or enabled.

npm run build

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/GP2-2936
- [x] Jira ticket has the correct status.
- [x] [Changelog](CHANGELOG.md) entry added.

### Reviewing help

- [ ] Explains how to test locally, including how to set up appropriate data
- [ ] Includes screenshot(s) - ideally before and after, but at least after

### Housekeeping

- [ ] Added all new environment variables to Vault.
- [ ] Cleaned up old feature flags
- [ ] Upgraded any vulnerable dependencies.
- [ ] Ran the `make manage download_geolocation_data` command
- [ ] I have updated security dependencies
- [ ] Python requirements have been re-compiled.
- [x] Frontend assets have been re-compiled.

### Merging

- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

File: react-components/src/components/Form/Radiogroup/RadiogroupItem.jsx [hunk 1]
```diff
@@ -1,11 +1,15 @@
 /* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
-import React, { memo } from 'react'
+import React, { memo, useState, useEffect } from 'react'
```
Comment (bobmeredith) [inline react-components/src/components/Form/Radiogroup/RadiogroupItem.jsx:2]: Looks like useState and useEffect are not being used.  I guess eslint will moan about this.

**Review by bobmeredith — CHANGES_REQUESTED** [review]
Need to remove unused imports.

File: react-components/src/components/Form/Radiogroup/RadiogroupItem.jsx (hunk 1 above)
Comment (seijihg) [reply react-components/src/components/Form/Radiogroup/RadiogroupItem.jsx]: fixed thanks for pointing this!

**Review by bobmeredith — APPROVED** [review]
LGTM

**Pull request merged**